lsp-mode.el

```diff
@@ -187,7 +187,7 @@ As defined by the Language Server Protocol 3.16."
      lsp-sorbet lsp-sourcekit lsp-sonarlint lsp-tailwindcss lsp-tex lsp-terraform
      lsp-toml lsp-ttcn3 lsp-typeprof lsp-v lsp-vala lsp-verilog lsp-vetur lsp-volar
      lsp-vhdl lsp-vimscript lsp-xml lsp-yaml lsp-ruby-lsp lsp-ruby-syntax-tree
-     lsp-sqls lsp-svelte lsp-steep lsp-tilt lsp-zig)
+     lsp-sqls lsp-svelte lsp-steep lsp-tilt lsp-zig lsp-credo)
```

Comment:
I think the list is stored in alphabetic order.

Reply:
yeah, you are right. I updated it.

Reply:
Thank you!